src/net.cpp

```diff
@@ -2247,7 +2247,14 @@ CNode::CNode(SOCKET hSocketIn, const CAddress& addrIn, const std::string& addrNa
     }
     idStr = tfm::format("%d", id);
 
-    ReloadTracingSpan();
```

Comment:
NACK, I think this is unnecessary noise, given that we would just be reverting this reversion immediately upon implementing the actual fix.

Reply:
OK, force-pushed to remove this.